### PR TITLE
QA | download path for chrome and firefox

### PR DESCRIPTION
**Pull request by bowka**
- new version of chromedriver 75 needs a bit different setting for download path.
- FF tweaks with download_path: saveToDisk all content types
- add method which can be easily called from test self.ts.download_path(). It returns variable which is set up in config.properties file
- add wait_for_file_to_be_downloaded() helpfull with combination with self.ts.download_path()

### CODE REVIEW

File: shishito/runtime/environment/shishito.py [hunk 1]
```diff
@@ -138,36 +139,28 @@ def add_cmdline_arguments_to_browser(self, browser_capabilities, config_section)
...
-        """
+    def set_download_path(self, driver):
+        """set download path for Chrome. Variable in config file download_path can be either string or env var."""
         try:
-            browser_name = self.shishito_support.get_opt(config_section, 'browser').lower()
-            download_file_path = self.shishito_support.get_opt('download_path')
-            m = re.match('^\$([A-Z][A-Z_]+)$', download_file_path)
-
+            download_directory = self.shishito_support.get_opt('download_path')
+            m = re.match('^\$([A-Z][A-Z_]+)$', download_directory)
```
Comment (vafliik) [inline shishito/runtime/environment/shishito.py]: If nothing is set in the `.properties` file, the `download_directory` is `None`, which will cause an Exception on this line  (`match()` does not accept `None`)

There should be a condition - if not setup in config file, ignore this completely

File: shishito/runtime/environment/shishito.py [hunk 2]
```diff
@@ -138,36 +139,28 @@ def add_cmdline_arguments_to_browser(self, browser_capabilities, config_section)
...
             if m is not None:
```
Comment (vafliik) [inline shishito/runtime/environment/shishito.py]: I am not sure about this whole `if` section. What is it trying to solve?

Comment (vafliik) [reply shishito/runtime/environment/shishito.py]: I get it that the whole regex thing is looking for env variable, if the config is e.g. like this:
`download_path=$DIRECTORY`.
But the whole logic for getting the env variables is already in the `shishito_support.get_opt()` method, no?

Comment (bowka) [reply shishito/runtime/environment/shishito.py]: nice I ddint know it is alreay implemented in `shishito_support.get_opt()`
anyway that regex was legacy code

**Pull request merged**